### PR TITLE
Task 12

### PR DESCRIPTION
**Pull request by evTyurin**

### CODE REVIEW

File: server/src/main/java/com/senlainc/warsaw/tyurin/service/OrderService.java [hunk 1]
```diff
@@ -64,6 +67,7 @@ public void shiftCompletionDateTime(long id, LocalDateTime completionDateTime) t
             orderDAO.getOrder(id).setCompletionDate(completionDateTime);
         } else {
             System.out.println("Shifting completion time was prohibited");
```
Comment (mrseneca) [inline server/src/main/java/com/senlainc/warsaw/tyurin/service/OrderService.java]: You don't need both. Use logger and get rid of SOUTs

**Review by mrseneca — CHANGES_REQUESTED** [review]
Well done, one small thing

**Comment by mrseneca** [comment]
Also, please, look up naming conventions for variables, classes and packages. You had plenty mistakes in it
Ask for assist if you can not find issues

File: server/src/main/java/com/senlainc/warsaw/tyurin/service/OrderService.java (hunk 1 above)
Comment (evTyurin) [reply server/src/main/java/com/senlainc/warsaw/tyurin/service/OrderService.java]: Done

**Comment by evTyurin** [comment]
> Also, please, look up naming conventions for variables, classes and packages. You had plenty mistakes in it Ask for assist if you can not find issues

I reread about naming for variables, classes and packages but still have problems with finding issues. It will be great if you lend me a hand in this case

File: server/src/main/java/com/senlainc/warsaw/tyurin/util/csvHandlers/CsvReader.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package com.senlainc.warsaw.tyurin.util.csvHandlers;
```
Comment (mrseneca) [inline server/src/main/java/com/senlainc/warsaw/tyurin/util/csvHandlers/CsvReader.java]: > The prefix of a unique package name is always written in all-lowercase ASCII letters and should be one of the top-level domain names, currently com, edu, gov, mil, net, org, or one of the English two-letter codes identifying countries as specified in ISO Standard 3166, 1981.

-> csvhandlers or csv.handlers if you have few subpackages

File: server/src/main/java/com/senlainc/warsaw/tyurin/util/dbConnection/DBConnector.java [hunk 3]
```diff
@@ -11,6 +12,8 @@
 @DependencyClass
 public class DBConnector {
```
Comment (mrseneca) [inline server/src/main/java/com/senlainc/warsaw/tyurin/util/dbConnection/DBConnector.java]: same here -> dbconnection

File: server/src/main/java/com/senlainc/warsaw/tyurin/util/jsonHandlers/JsonReader.java [hunk 4]
```diff
@@ -11,6 +12,9 @@
 
 @DependencyClass
 public class JsonReader {
```
Comment (mrseneca) [inline server/src/main/java/com/senlainc/warsaw/tyurin/util/jsonHandlers/JsonReader.java:14]: same here -> jsonhadlers

File: server/src/main/java/com/senlainc/warsaw/tyurin/dao/CraftsmanDAO.java [hunk 5]
```diff
@@ -6,16 +6,20 @@
...
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 @DependencyClass
 public class CraftsmanDAO implements ICraftsmanDAO{
```
Comment (mrseneca) [inline server/src/main/java/com/senlainc/warsaw/tyurin/dao/CraftsmanDAO.java]: > Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

Usually in production development reviewers asks to change to Dao, it's a matter of taste but still a thing so
-> CraftsmanDao

File: server/src/main/java/com/senlainc/warsaw/tyurin/dao/CraftsmanDAO.java [hunk 6]
```diff
@@ -6,16 +6,20 @@
...
 
+    private final static Logger logger = Logger.getLogger(CraftsmanDAO.class);
+
     @DependencyComponent
     private DBConnector dbConnector;
```
Comment (mrseneca) [inline server/src/main/java/com/senlainc/warsaw/tyurin/dao/CraftsmanDAO.java]: Same here -> DbConnector

**Review by mrseneca — CHANGES_REQUESTED** [review]
Here oracle page for JNC
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

**Comment by evTyurin** [comment]
Thank you. I update naming for classes and packages you showed me

**Review by mrseneca — APPROVED** [review]
Well done

Check and fix other packages with next PR, I can see there still some 
изображение

**Pull request merged**